buildbot_nix/__init__.py

```diff
@@ -276,10 +276,10 @@ async def schedule(
         for brid in brids.values():
             url = getURLForBuildrequest(self.master, brid)
             await self.addURL(f"{scheduler.name} #{brid}", url)
-            self._add_results(brid)
+            await self._add_results(brid)
```

Comment:
This is now also in main.

Reply:
Will revert, i just needed pylsp to shur up